components/layout/table_row.rs

```diff
@@ -64,14 +64,14 @@ pub struct TableRowFlow {
     pub collapsed_border_spacing: CollapsedBorderSpacingForRow,
 }
 
-impl Encodable for TableRowFlow {
-    fn encode<S: Encoder>(&self, e: &mut S) -> Result<(), S::Error> {
-        self.block_flow.encode(e)
+impl Serialize for TableRowFlow {
+    fn serialize<S: Serializer>(&self, serializer: &mut S) -> Result<(), S::Error>{
```

Comment:
nit: space before { (there are a bunch of other instances in this PR; search for `>{`)